javascript-typing-tutor/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Typing Tutor</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Arvo:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="center">
+    <div class="color word-style">
+      <span>g</span><span>r</span><span>u</span><span>m</span><span>p</span><span>y</span>
+      <span> </span>
```

Comment:
If you move the empty space span to the previous line it will underline as you type like the rest of the characters